.atomist/handlers/command/reaction/ReactIssue.ts

```diff
@@ -23,8 +24,19 @@ class ReactIssueCommand implements HandleCommand {
     corrid: string
 
     handle(ctx: HandlerContext): Plan {
-        let plan = new Plan();
-        let execute = {instruction: {kind: "execute", name: "react-github-issue", parameters: this}};
+        const ghRepo = new Repository(this.owner, this.repo, `#{github://user_token?scopes=repo}`)
+        const ghIssue = ghRepo.issue(Number(this.issue))
+        const http = ghIssue.react({ "content": <ReactionContent> this.reaction })
+
+        const plan = new Plan();
```

Comment:
We could probably use some functions to simplify all the plan, instruction and response handler stuff too!